package.json

```diff
@@ -4,17 +4,18 @@
   "description": "",
   "main": "node server.js",
   "scripts": {
-    "dev": "node src/server.js",
-    "dev-watch": "nodemon src/server.js",
-    "build": "rimraf build && cpx \"src\\**\\*\" build && next build ./build",
+    "dev": "ts-node --type-check --compiler-options '{\"module\":\"commonjs\"}' src/server.ts",
+    "dev-watch": "nodemon src/server.ts",
+    "build": "rimraf build nextbuild && cpx \"src/**/*\" nextbuild && next build ./nextbuild && tsc --project tsconfig.server.json && cpx \"nextbuild/.next/**/*\" build/.next && cpx \"src/static/**/*\" build/static/",
     "start": "NODE_ENV=production node build/server.js",
     "now-start": "NODE_ENV=now node build/server.js",
     "test": "NODE_ENV=test jest",
     "test-sequential": "jest --runInBand",
-    "lint": "eslint --ext .js,.jsx ./src",
-    "fix-lint": "npm run lint -- --fix",
-    "prettier": "prettier --write \"src/**/*.{js,jsx}\"",
-    "prettier-diff": "prettier --list-different \"src/**/*.{js,jsx}\"",
+    "lint-js": "eslint --ext .js,.jsx ./src",
+    "fix-lint-js": "npm run lint-js -- --fix",
```

Comment:
Maybe we could add a `lint-ts` target for TypeScript linting with [typescript-eslint](https://github.com/typescript-eslint/typescript-eslint)?